hardware/system/upboard/install_script.sh

```diff
@@ -1,19 +1,32 @@
 #!/bin/bash
 set -e
 
+TAG=$1
+MACHINE=$2
+
+if [ -z $1 ] || [ -z $2 ]; then
+  echo 'usage: install_script <machine> <version>'
+  exit 1
+fi
+
+if [ "$MACHINE" != "gaia" ] && [ "$MACHINE" != "sintra" ]; then
+  echo 'Install script expects "gaia" or "sintra" as machine parameter'
+  exit 1
+fi
+
 sudo apt update && sudo apt full-upgrade -y
 
 # install dependencies
 sudo apt install build-essential chromium curl git supervisor yasm -y
-sudo apt install libavcodec-dev libavformat-dev libswscale-dev -y
+sudo apt install libavcodec-dev libavformat-dev libswscale-dev libpcsclite-dev -y
```

Comment:
Why do we need ``libpcsclite-dev`` (or any of these)?

Reply:
Good point, I think I can remove all audio related stuff from the code today